apps/ejabberd/src/ejabberd_router.erl

```diff
@@ -95,11 +95,15 @@ start_link() ->
 %% the next module in sequence.
 -spec route(From   :: ejabberd:jid(),
     To     :: ejabberd:jid(),
-    Packet :: jlib:xmlel()) -> ok.
+    Packet :: mongoose_acc:t()|jlib:xmlel()) -> mongoose_acc:t().
+route(From, To, #xmlel{} = Packet) ->
+%%    ?ERROR_MSG("Deprecated - it should be Acc: ~p", [Packet]),
```

Comment:
Because otherwise it would clutter the shell :) The strategy is to uncomment it at the last stage of the work, to get rid of remaining calls.